### PR TITLE
added pgbackups list command

### PR DESCRIPTION
**Pull request by jdx**
All of the other PRs are based on this one for pgbackups

### CODE REVIEW

**Comment by jdx** [comment]
@uhoh-itsmaciek I'm going to be out in Nepal from 10/12-10/28 so I don't need all of these reviewed right away. However if you could review this one PR it would clean up the diffs of all the other backups commands.

File: commands/backups/index.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      cli.log(`No backups. Capture one with ${cli.color.cmd('heroku pg:backups:capture')}`)
+    } else {
+      cli.table(backups, {
+        columns: [
+          {label: 'ID', format: (_, t) => cli.color.cyan(pgbackups.transfer.name(t))},
+          {label: 'Created at', key: 'created_at'},
+          {label: 'Status', format: (_, t) => pgbackups.transfer.status(t)},
+          {label: 'Size', key: 'processed_bytes', format: b => pgbackups.filesize(b)},
+          {label: 'Database', key: 'from_name', format: d => cli.color.configVar(d) || 'UNKNOWN'}
+        ]
```
Comment (msakrejda) [inline commands/backups/index.js:34]: Very nice.

File: index.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+const flatten = require('lodash.flatten')
+
 exports.topics = [
   {name: 'pg', description: 'manage postgresql databases'}
 ]
 
-exports.commands = [
-  require('./commands/info').info,
-  require('./commands/info').root,
+exports.commands = flatten([
```
Comment (msakrejda) [inline index.js:9]: Nice, I should adapt `kafka` to do this too.

**Comment by msakrejda** [comment]
+1. Enjoy Nepal!

**Review by msakrejda — APPROVED** [review]

**Pull request merged**